src/applications/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -40,9 +40,9 @@ class VeteranIDCard extends React.Component {
         if (serviceRateLimitedAuthed) {
           recordEvent({ event: 'vic-authenticated-ratelimited' });
           this.renderEmailCapture = true;
-          if (userProfile.veteranStatus === 'NOT_FOUND') {
+          if (userProfile.veteranStatus.status === 'NOT_FOUND') {
```

Comment:
Probably not needed based on how you are giving `veteranStatus` a default value of `{}`, but to be extra careful, should likely do `userProfile.veteranStatus?.status` here...

Reply:
Done!